test/unit/sdk_reporter/test_sdk_reporter.py

```diff
@@ -20,11 +19,12 @@
 from codeguru_profiler_agent.codeguru_client_builder import CodeGuruClientBuilder
 
 profiling_group_name = "test-ProfilingGroup-name"
-lambda_one_click_profiling_group_name = "aws-lambda-testLambdaName"
-test_agent_metadata_for_lambda = AgentMetadata(
-    fleet_info=AWSLambda("arn:aws:lambda:us-east-1:111111111111:function:testLambdaName", "memory", "env", "agentId"))
+test_lambda_profiling_group_name = "aws-lambda-testLambdaName"
```

Comment:
Could this be renamed to something clearer e.g. autocreated_profiling_group_name_lambda or something like it?